src/transactions/SignatureChecker.cpp

```diff
@@ -37,6 +40,15 @@ SignatureChecker::checkSignature(AccountID const& accountID,
         return true;
     }
 
+	// flag any signatures from the whitelist as used
```

Comment:
why do they need to be flagged as used?

Reply:
The same reason other used signatures are marked as such.  It would be nice if Stellar had bothered to explain in the first place, but I don't want to focus on deficiencies in the inherited code.